crates/biome_cli/tests/snapshots/main_commands_check/apply_bogus_argument.snap.new

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
You have to execute `cargo insta accept` to turn `.snap.new` into `.snap`

Reply:
Yeah, Thanks. I've update that to change `.snap.new` to `.shap`.